zerver/openapi/zulip.yaml

```
@@ -9023,6 +9026,9 @@ paths:
       description: |
         This endpoint unmutes a user.
 
+        **Changes**: Before Zulip 8.0 (feature level 188), this endpoint does not allow
+        unmuting bot users, indicated by returning an error with a "No such user" message.
```

The specific error message shouldn't be in the documentation, except as part of an example. That's because it will vary with translations, and so client code should never attempt to inspect it. (When there's information that client code is expected to want to extract from an error, we express it machine-readably in `code` and possible additional fields.)

Combining with the other comments above, this could become:

```suggestion
        **Changes**: Before Zulip 8.0 (feature level 188), it was an error to specify a bot user.
```

---

That makes sense, thanks!